net.solarnetwork.node.weather.ibm.wc.test/src/net/solarnetwork/node/weather/ibm/wc/test/WCClientTests.java

```diff
@@ -0,0 +1,76 @@
+package net.solarnetwork.node.weather.ibm.wc.test;
+
+import java.util.Collection;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import net.solarnetwork.node.domain.GeneralAtmosphericDatum;
+import net.solarnetwork.node.domain.GeneralDayDatum;
+import net.solarnetwork.node.weather.ibm.wc.WCDayDatumDataSource;
+import net.solarnetwork.node.weather.ibm.wc.WCHourlyDatum;
+import net.solarnetwork.node.weather.ibm.wc.WCHourlyDatumDataSource;
+
+/**
+ * Tests retrieval from the IBM API
+ * 
+ * @author matt frost
+ */
+public class WCClientTests {
+	
+	
+	private static final String TEST_SOURCE_ID = "Test";
+
+	private WCDayDatumDataSource dailyService;
+	private WCHourlyDatumDataSource hourlyService;
+
+	private final Logger log = LoggerFactory.getLogger(getClass());
+
+	@Before
+	public void setup() {
+		dailyService = new WCDayDatumDataSource();
+		hourlyService = new WCHourlyDatumDataSource();
+	}
+
+	//@Test
+	public void readDailyDatum() {
+		dailyService.setLocationIdentifier("NZAA");
+		dailyService.setLanguage("en-US");
+		dailyService.setApiKey("b20225e17d6642598225e17d668259d2");
```

Comment:
This looks like a real API key, which shouldn't be checked into git. This test should not actually make HTTP calls to a real web service URL. It should start up an embedded HTTP server that the test can control to return JSON resources similar to what you've done in other tests.

Use the [Yr client tests](https://github.com/SolarNetwork/solarnetwork-node/blob/1fb37e491ce0111b2c33004d640c97595087d4e1/net.solarnetwork.node.weather.yr.test/src/net/solarnetwork/node/weather/yr/test/XmlYrClientTests.java#L71-L85) for reference on how to do this.